apps/addons/fixtures/addons/featured.json

```diff
@@ -263,14 +262,12 @@
         "model": "files.file",
         "fields": {
             "status": 4,
-            "codereview": 0,
             "hash": "sha256:5b5a",
             "created": "2007-03-05 13:19:15",
             "modified": "2007-04-04 12:30:11",
             "filename": "del.icio.us_bookmarks-1.0.43-fx.xpi",
             "platform": 1,
             "version": 1003,
-            "modified": "2007-04-04 12:30:11",
```

Comment:
It's a duplicate of line 267:

```
             "modified": "2007-04-04 12:30:11",
```

See https://github.com/mozilla/olympia/pull/661/files#diff-11ddc2768fbda00caaf8145fa453fb25R267